dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceFormatter.java

```diff
@@ -1106,7 +1091,7 @@ protected String formatStreamOutput(GenericData genData) {
             }
         }
 
-        String message = filterStackTraces(txt);
+        String message = BaseTraceService.filterStackTraces(txt);
```

Comment:
Since you switch to the BaseTraceSerice.filterStackTraces(txt), do we still need to keep the private method filterStackTraces(String txt) in this class at line 1103?

Reply:
Hi Felix, no it's not necessary. I've made the necessary change to remove the redundant method. Thanks!